knowledge_base/save_job_result_to_volume/resources/top_ten_trips.job.yml

```diff
@@ -0,0 +1,26 @@
+resources:
+  jobs:
+    top_ten_trips:
+      name: top_ten_trips
+
+      trigger:
+        # Run this job every day to update all time the top ten trips recorded top
+        # 10 trips in the UC Volume.
+          interval: 1
+          unit: DAYS
+
+      # No job cluster is configured. The job will run on serverless compute. 
+      # You can explicitly configure job compute here if your workspace does
+      # not have serverless compute enabled.
+      tasks:
+        - task_key: top_ten_trips_task
+          notebook_task:
+            notebook_path: ../src/query.ipynb
+
+      parameters: 
+        - name: schema_name
+          default: ${resources.schemas.trips.name}
```

Comment:
Can you synthesize the volume path here?

I.e. `/Volumes/catalog/schema/volume`.

Then the path can be used directly.

Reply:
done